.github/CODEOWNERS

```diff
@@ -0,0 +1,10 @@
+# https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax
+#
+# These owners will be the default owners for everything in
+# the repo, unless a later match takes precedence.
+* @madskristensen
+
+# Managed by Ansible DevTools Team members:
+src/schemas/json/ansible-* @ssbarnea @webknjaz
```

Comment:
@madskristensen FYI GH does not notify accounts that are not added to the org or the repo collaborators. To make this work, you may want to create teams with read or triage privileges.